example-code/snippets/src/backend/IModelBranchingOperations.test.ts

```diff
@@ -31,7 +32,7 @@ async function initializeBranch(myITwinId: string, masterIModelId: string, myAcc
   const masterDb = await BriefcaseDb.open({ fileName: masterDbProps.fileName });
 
   // create a duplicate of master as a good starting point for our branch
-  const branchIModelId = await IModelHost.hubAccess.createNewIModel({
+  const branchIModelId = await IModelHost[_hubAccess].createNewIModel({
```

Comment:
We can't use internal APIs like `IModelHost[_hubAccess]` in the published parts of example code snippets. It appears core-backend does not expose a public API to create a new iModel.

Reply:
Should I add createNewIModel as a new public static function on IModelHost?

Reply:
I added `createNewIModel` as a public static function on `IModelHost`.